test/parallel/taskCompare/elliot/empty-omp-parallel-for.skipif

```diff
@@ -0,0 +1,6 @@
+# skip configs where -fopenmp or /proc/cpuinfo aren't supported
+CHPL_TARGET_COMPILER==clang
+CHPL_TARGET_COMPILER<=pgi
+CHPL_TARGET_COMPILER<=cray
+CHPL_TARGET_PLATFORM>=linux
```

Comment:
Actually nevermind, it won't run on crays anyways because we skip C tests when `CHPL_LAUNCHER != none`